pymeasure/instruments/__init__.py

```diff
@@ -60,6 +60,7 @@
 from . import novanta
 from . import oxfordinstruments
 from . import parker
+from . import philips
```

Comment:
In a recent change, we do not include manufacturers in this file anymore.

So, please remove the line added to this file.

```suggestion
```